### PR TITLE
Updated test-prow-e2e file to accommodate ceph

### PR DESCRIPTION
**Pull request by cloudbehl**
Signed-off-by: Ankush Behl <cloudbehl@gmail.com>

### CODE REVIEW

**Review by spadgett — APPROVED** [review]
/lgtm

File: test-prow-e2e.sh [hunk 1]
```diff
@@ -34,7 +34,7 @@ export DBUS_SESSION_BUS_ADDRESS
 
 SCENARIO="${1:-e2e}"
 
-if [ "$SCENARIO" != "login" ]; then
+if [ "$SCENARIO" != "login" ] || ["$SCENARIO" != "ceph"] ; then
```
Comment (spadgett) [inline test-prow-e2e.sh:37]: Doesn't this need to be `&&`?

**Review by spadgett — CHANGES_REQUESTED** [review]
/lgtm cancel

File: test-prow-e2e.sh (hunk 1 above)
Comment (cloudbehl) [reply test-prow-e2e.sh]: Ahh right!

**Review by spadgett — APPROVED** [review]
/lgtm

**Pull request merged**